packages/svgo-jsx/svgo-jsx.js

```diff
@@ -162,7 +174,12 @@ export const convertSvgToJsx = ({
     throw Error(error);
   }
   try {
-    return convertXastToJsx(xast, target, svgProps);
+    const components = [];
```

Comment:
Maybe will be better to use `Set` here to avoid duplicate components?

Reply:
Good catch, I'll add this case to test